spec/sequencescape/api/v2/tube_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+class SomeStockPlates
+  def initialize(stock_plates)
+    @stock_plates = stock_plates
+  end
+
+  def where(_arg)
+    @stock_plates
+  end
+end
+
+RSpec.describe Sequencescape::Api::V2::Tube do
+  subject(:tube) { create :v2_tube, barcode_number: 12_345 }
+
+  it { is_expected.to_not be_plate }
+  it { is_expected.to be_tube }
+
+  describe '#stock plate' do
+    let(:stock_plates)          { create_list(:v2_stock_plate, 4) }
+    let(:tube_with_ancestors)   { create :v2_tube, barcode_number: 12_345, ancestors: stock_plates }
+
+    it 'should return the last plate' do
+      expect(stock_plates.length).to eq(4)
+    end
+
+    # I know this is a real hack but all we need to know is whether
+    # it returns the last stock plate
+    # I am not going to fumble about trying to recreate the whole pipeline
+    it 'should have some ancestors' do
```

Comment:
Feel like you've got your test names swapped around? This one should say 'should return the last plate' and the other one 'should have some ancestors' ?

Reply:
It only needs one test so will remove the other one.